index.js

```diff
@@ -18,6 +18,7 @@ export default function ({types: t}) {
   const processChildren = (children) => {
     return children.map(c => {
       if (t.isJSXElement(c) || t.isStringLiteral(c) || t.isJSXExpressionContainer(c)) {
+        if (t.isStringLiteral(c)) return t.JSXText(c.value);
```

Comment:
Sure, I wondered if this was a bit too general. We found a component yesterday that this fix didn't work on so hopefully it'll take care of that too